### ISSUE TITLE
wfb-ng Download configuration problems to Ubuntu22.04

### ISSUE THREAD

**Issue by dangbo08** [svpcom/wfb-ng#379]
May I ask, I configured wfb-ng on Ubuntu22.04, and downloaded the driver of the network adapter, I want to use the Ubuntu virtual machine as a ground station to receive openipc camera data, although my network adapter received pkt, but I checked, there is no data in the pkt, what is going on? Can someone tell me, thank you very much!!
![QQ截图20241025114520](https://github.com/user-attachments/assets/d8b11c8e-b515-4d80-9d35-61e06e4727ca)

**Comment by dangbo08** [comment]
I generated dron.key and gs.key in the openipc camera, then gave the gs.key to the Ubuntu ground station, and configured the WiFi channel to be the same, but it still didn't work.

**Comment by svpcom** [comment]
Try to update GS to the latest version. It may be version incompatibility between drone and gs

**Comment by dangbo08** [comment]
![new](https://github.com/user-attachments/assets/f9db4618-868b-4e0b-9b2b-ee08cc7080ab)
Hello, I downloaded the new wfb-ng, but the result is still the same, I would like to ask whether wfb-ng must correspond to its version? Or is it backward compatible? Because I downloaded the ground station firmware from openipc's official website, burning it into nvr can connect with my openipc camera and display images.
![IMG_20241028_102111](https://github.com/user-attachments/assets/65ac5bbd-094f-4469-b0da-0e507f01c507)
![IMG_20241028_102122](https://github.com/user-attachments/assets/e116883a-a689-4577-92ee-1927242b02f9)

**Comment by dangbo08** [comment]
![new22](https://github.com/user-attachments/assets/64b4cd7f-5c0a-4aec-805c-fef344dcfcd6)
Hello, I have been able to display pictures, but the delay and packet loss problems are very serious, is this situation related to my WiFi network card?

**Comment by svpcom** [comment]
yes. A lot of lost packets shows that there is a problem with wifi card and/or usb bus. Do you use native linux or run it in the virtual machine?

**Comment by dangbo08** [comment]
I ran this on a virtual machine.

**Comment by svpcom** [comment]
This is a known problem described in SetupHOWTO. Use native linux
